langchain/src/client/langchainplus.ts

```diff
@@ -1,94 +1,18 @@
-import { BaseRun, Run, RunType } from "../callbacks/handlers/tracer.js";
+import { Example, LangChainPlusClient } from "langchainplus-sdk";
```

Comment:
What's the rationale for keeping some methods in the main framework but not others? Why not move all these into the new SDK?

Reply:
The main rationale is that the SDK doesn't depend on any LangChain primitives, so things like "chains", "llms", etc. aren't accessible (unless we add duplicate typing stubs/schema for the BaseLanguageModel, Chain, etc. to the SDK).

So then when running in LangChain, the most straightforward things to do would either be:
- Standalone functions (current proposal)
- Extended client

Between the latter two, I don't have a strong preference but it's not a 1-way door